packages/ui/src/components/DateInput/__stories__/Range.stories.tsx

```diff
@@ -1,25 +1,69 @@
 import type { StoryFn } from '@storybook/react'
+import type { ComponentProps } from 'react'
 import { useState } from 'react'
 import { DateInput } from '..'
+import { Stack } from '../../Stack'
 
-export const Range: StoryFn = args => {
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+]
+export const Range: StoryFn<ComponentProps<typeof DateInput>> = args => {
```

Comment:
There is a small issue on range version, when no dates are selected we have `-` while it shouldn't

![Screenshot 2024-11-13 at 11 09 51](https://github.com/user-attachments/assets/169458f7-9f3e-490b-a7fe-da997051163d)